src/tempstorage/TemporaryStorage.py

```diff
@@ -298,6 +298,7 @@ def _finish(self, tid, u, d, e):
         self._tmp = []
 
     def _takeOutGarbage(self, oid):
+        return
```

Comment:
Is this deliberate or a leftover? If it's deliberate, why not delete the method?